src/Helpers/Arr.php

```diff
@@ -4,7 +4,7 @@
 
 class Arr
 {
-    public static function map(array $array, $callback): array
+    public static function map(array $array, callable $callback): array
```

Comment:
Strictly speaking `array_map` also accepts null, but `callable` will do for 99.99% of the cases.